dotnet/aspnet/mssqldb/Application/SampleWebApplication.FunctionalTests/SampleWebApplication.FunctionalTests.csproj

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7D15A631-042A-4C9D-BA57-FF60AA7F48C8}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>SampleWebApplication.FunctionalTests</RootNamespace>
+    <AssemblyName>SampleWebApplication.FunctionalTests</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Is it same as the target framework version of the web app?

Reply:
check other places too

Reply:
Pasted the wrong folder while creating a new branch. Fixing this in the next commit in a few mins

Reply:
Fixed